.travis.yml

```diff
@@ -30,8 +30,8 @@ before_install:
     - mkdir -p $SPLUNK_HOME/var/log/splunk
 
 env:
-    - SPLUNK_VERSION=6.2.6-sdk
-    - SPLUNK_VERSION=6.3.1-sdk
+    - SPLUNK_VERSION=6.4-sdk
+    - SPLUNK_VERSION=6.5-sdk
```

Comment:
Yeah but they shouldn't be in this PR since they're on `develop`. @gaurav22gupta can you re-target this PR to the develop branch?